source/extensions/omni.isaac.lab/test/sensors/test_camera.py

```diff
@@ -360,6 +360,50 @@ def test_intrinsic_matrix(self):
             torch.testing.assert_close(rs_intrinsic_matrix[0, 0, 0], camera.data.intrinsic_matrices[0, 0, 0])
             # torch.testing.assert_close(rs_intrinsic_matrix[0, 1, 1], camera.data.intrinsic_matrices[0, 1, 1])
 
+    def test_depth_clipping(self):
+        """Test depth clipping.
+
+        .. note::
+
```

Comment:
How are we forcing `NaN` or infinity values in these tests?

Reply:
What do you mean by forcing these values? `NaN` of `inf` are values that arise if no point is hit but vary on the sensor implementation which one is generated

Reply:
I meant in order to test that clipping works well, I thought we would need to make it so that there were `NaN` / `inf` values from the scene. Is this guaranteed based on the scene?

Reply:
Ah, yes we have these values by design of sensor and scene